storage-proofs-porep/src/stacked/vanilla/proof.rs

```diff
@@ -59,6 +60,8 @@ lazy_static! {
     /// It might be possible to relax this constraint, but in that case, only one builder
     /// should actually be active at any given time, so the mutex should still be used.
     static ref GPU_LOCK: Mutex<()> = Mutex::new(());
+
+    static ref THREAD_POOL: Pool = Pool::new(num_cpus::get());
```

Comment:
Here you are using `num_cpus::get()`whereas in `bellperson` you are using an env-var `BELLMAN_NUM_CPUS` to get the number of CPU's, should these two values always agree?

Reply:
No, I don't think so.  The bellman var is bellman specific, and that's ok.  If we had a proofs specific one, it might be different, but proofs generally targets max parallelism from the available cores.